providers/src/airflow/providers/fab/www/templates/swagger-ui/index.j2

```diff
@@ -0,0 +1,87 @@
+{#
```

Comment:
Copy of `airflow/www/templates/swagger-ui/index.j2`

Reply:
hmm, do we need the swagger ui stuff still? That's only the api reference, right?

Reply:
Good point! I removed it and did a bit of cleanup in the `webpack.config.js`. I am pretty sure we could remove more stuff but that's something we can do later